visual-studio-dev-vm-O365/azuredeploy.json

```diff
@@ -1,242 +1,242 @@
+      }
+    },
+    "vmVisualStudioVersion": {
+      "type": "string",
+      "defaultValue": "VS-2015-Ent-VSU3-AzureSDK-291-Win10-N",
+      "allowedValues": [
+          "VS-2015-Comm-AzureSDK-2.9-W10T-Win10-N",
+          "VS-2015-Comm-AzureSDK-2.9-WS2012R2",	
+          "VS-2015-Comm-VSU3-AzureSDK-291-Win10-N",	
+          "VS-2015-Comm-VSU3-AzureSDK-291-WS2012R2",	
+          "VS-2015-Community-AzureSDK-2.7-Cordova-Win8.1-N-x64",	
+          "VS-2015-Community-AzureSDK-2.7-W10T-Win10-N",
+          "VS-2015-Community-AzureSDK-2.7-WS2012R2",
+          "VS-2015-Ent-AzureSDK-2.9-WS2012R2",
+          "VS-2015-Ent-AzureSDK-29-W10T-Win10-N",
+          "VS-2015-Ent-VSU3-AzureSDK-291-Win10-N",	
+          "VS-2015-Ent-VSU3-AzureSDK-291-WS2012R2",
+          "VS-2015-Enterprise-AzureSDK-2.7-Cordova-Win8.1-N-x64",
+          "VS-2015-Enterprise-AzureSDK-2.7-W10T-Win10-N",
+          "VS-2015-Enterprise-AzureSDK-2.7-WS2012R2",
+          "VS-2015-Pro-AzureSDK-2.8-Cordova-Win8.1-N-x64",	
+          "VS-2015-Professional-AzureSDK-2.7-Cordova-Win8.1-N-x64",	
+          "VS-2015-Professional-AzureSDK-2.7-W10T-Win10-N"
+      ],
+      "metadata": {
+        "description": "Which version of Visual Studio (MSDN Subscription required) you would like to deploy"
+      }
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+      "description": "DNS Label for the Public IP. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "officeVersion": {
+      "type": "string",
+      "defaultValue": "Office2016",
+      "allowedValues": [
+        "Office2016",
+        "Office2013"
+      ],
+       "metadata": {
+        "description": "Which version of Office would you would like to deploy"
+      }
+    }    
+  },
+  "variables": {
+    "storageAccountName": "[concat(uniquestring(resourceGroup().id), 'visstudiovm')]",
+    "vnet01Prefix": "10.0.0.0/16",
+    "vnet01Subnet1Name": "Subnet-1",
+    "vnet01Subnet1Prefix": "10.0.0.0/24",
+    "vmImagePublisher": "MicrosoftVisualStudio",
+    "vmImageOffer": "VisualStudio",
+    "vmOSDiskName": "VMOSDisk",
+    "vmVnetID": "[resourceId('Microsoft.Network/virtualNetworks', 'Vnet01')]",
+    "vmSubnetRef": "[concat(variables('VMVnetID'), '/subnets/', variables('Vnet01Subnet1Name'))]",
+    "vmStorageAccountContainerName": "vhds",
+    "vmNicName": "[concat(variables('vmName'), 'NetworkInterface')]",
+    "vmIP01Name": "VMIP01",
+    "storageType": "Premium_LRS",
+    "vmSize": "Standard_DS2",
+    "setupOfficeScriptUri": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/visual-studio-dev-vm-O365/scripts/",

```

Comment:
please see the best practices guide about _artifactsLocation: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md